project/new-evaluation.json

```diff
@@ -0,0 +1,213 @@
+{"fieldsets":
+    [{
+        "items": [
+            {
+                "label": "Classes",
+                "options": [ "0", "1", "2", "3" ]
```

Comment:
Since all of these `option`s get repeated for each of the config objects, we might want to abstract them away into a separate property to avoid repeating the same data throughout this file.

Reply:
no idea how to solve this. sounds like a requestception :O

Reply:
What if our config object had a separate top-level property, something like `options` or `grading` or `grades` or something of the sort, that we could use in the rendering methods instead of the ones defined for each item in part? E.g.:
```javascript
let config = {
    "options": ["0", "1", "2", "3"],
    "items": [
        {
            "label": "Classes"
        },
        ...
    ]
}
// then, later on...
container.innerHTML = config.items.map(renderItem).join('');
// where renderCategory uses the values in config.options to render each of the <option> elements
```

Reply:
o sa incerc. chiar sunt curioasa sa vad daca iese ceva. mersi de idee :D

Reply:
Sure thing! Anytime! ;)